steps/triggers/triggers.go

```diff
@@ -25,5 +31,9 @@ var _ = gauge.Step("Cleanup Triggers", func() {
 })
 
 var _ = gauge.Step("Mock post event to <interceptor> interceptor with event-type <eventType>, payload <payload>", func(interceptor, eventType, payload string) {
-	gauge.GetScenarioStore()["response"] = triggers.MockPostEvent(store.GetScenarioData("route"), interceptor, eventType, payload)
+	gauge.GetScenarioStore()["response"] = triggers.MockPostEvent(store.GetScenarioData("route"), interceptor, eventType, payload, false)
+})
+
+var _ = gauge.Step("Mock TLS post event to <interceptor> interceptor with event-type <eventType>, payload <payload>", func(interceptor, eventType, payload string) {
```

Comment:
why do we require another step for it, can we add a parameter to spec file itself??

Reply:
modified